src/settings_gui.cpp

```diff
@@ -859,9 +877,14 @@ struct GameOptionsWindow : Window {
 				for (const auto &setting_type : setting_types) {
 					size.width = std::max(size.width, GetStringBoundingBox(GetString(STR_CONFIG_SETTING_TYPE, setting_type)).width + padding.width);
 				}
+				size.height = 2 * GetCharacterHeight(FS_NORMAL);
```

Comment:
`NUM_PROPERTY_LINES` instead of 2, perhaps? (To match below)

Reply:
The number only appears once here. There are plenty of other numbers in the other `case`. I don't think there is any benefit in adding more constants.